src/reducers/splitView.js

```diff
@@ -0,0 +1,9 @@
+
+export const updateSplitPosition = (state, { value }) => {
+  localStorage.setItem('splitPosition', parseInt(value, 10))
```

Comment:
There shouldn't be any side effects in reducers. I believe this should go in the action-creator thunk.

Reply:
Thanks. I'll change it